service/controller/v12/resource/vpn/template/main.json

```diff
@@ -38,7 +38,7 @@
         "provider":"[toUpper(parameters('GiantSwarmTags').provider)]"
       },
       "sku":{
-        "name": "Basic"
+        "name": "Standard"
```

Comment:
See comment above for the default behavior (when multi-AZ parameters are not specified at all).
It would be good to keep using `Basic` load balancer by default, when multi-AZ params are not specified - one reason would be because `Basic` LB is free, while `Standard` is paid, so we could try to keep the costs down (unless we have other reasons to use `Standard` LB).

Reply:
I'm curious how is it `Basic` still here? I was under impression we moved to `Standard` before due to various networking issues with `Basic` one. @T-Kukawka do you have more insight in here?